.circleci/config.yml

```diff
@@ -134,13 +134,11 @@ jobs:
       - run:
           name: 'Check if any unit tests failed'
           command: |
-            FAILURES=$(find reports -iname 'webex*.xml' -print0 | xargs -0 sed -n 's/.*failures=\"\([^\"]*\).*/\1/p' | xargs)
-            ERRORS=$(find reports -iname 'webex*.xml' -print0 | xargs -0 sed -n 's/.*errors=\"\([^\"]*\).*/\1/p' | xargs)
```

Comment:
Why are we no longer checking ERRORS?

Reply:
It was a while ago, but one of Arun's PR's had passed all the tests but the  job itself failed because mocha/karma had reported an error. It didn't report what the error was so to me it was just a false positive for a passing test suite and an error is more on the runner side than the test side